### DISCUSSION TITLE
I compiled from source, works fantastically

### DISCUSSION THREAD

**Discussion by awesome14** [qarmin/czkawka#665] [General]
I just wanted to give the developers 10 kudos each, because this is one of the best file utilities I've seen, and that's going back to the 1990s. On one Linuix machine that hasn't seen much cleanup, czkawka founf 97,000+ duplicates by hash. Random manual checking using md5sum demonstrated it was statistically 100% accurate!

That process required ~00:5:00, and deletion took a bit longer. Maybe you're using system tools for deletion instead of kernel calls. I am very grateful to the developers for all the hard work to produce this gem! Thank you!

**Comment by charlesIsOftenWrong** [comment]
Similar experience!  I was able to compile from source, in spite of the very vague instructions that seem to be vague do to the limitations of the writer's English language skills.  I have ZERO knowledge or experience with RUST, so much of the instructions were foreign to me. 
 I still give the dude (chica?) a solid A for effort!  The use of English may be horrific, but they **MADE THE EFFORT**  and got the essential information conveyed.  In spite of my unfamiliarity with the RUST environment, I was able to figure it out get a clean build to my liking.

The application itself seems to be a solid clone of fslint, which I have used for many years.  Thus far it seems to be a solid expansion of fslint with an updated interface and new features.  NICE!

I absolutely love seeing new contributions to opensource like czkawka!  Yall (texas, second person plural, not standard english) seem to be doing a great job and a great service to the spirit of gnu!  **BIG TEXAS THANKS!**
